webpack/webpack.common.js

```diff
@@ -21,19 +21,12 @@ module.exports = {
           },
           {
             loader: "ejs-loader",
+            options: {
+              variable: "data",
```

Comment:
I don't 100% understand the need for this change (and the associated changes), but I can guess what it does based on reading https://github.com/difelice/ejs-loader#options

Reply:
OK, my guess wasn't really accurate, the commit message gives a better hint: `Add data variable to ejs loader for jst templates, to avoid errors in strict mode`

Reply:
better explanation from @ksy36:

> ejs loader adds `with()` for variables that are passed to the template, which errors out in strict mode

Now i get it. 👍